services/index.ts

```diff
@@ -33,6 +33,20 @@ const validateRequest = (req, res, next) => {
     next()
 }
 
-app.use('/graphql', bodyParser.json(), validateRequest, graphqlExpress({ schema }))
+app.use( '/graphql',
+    bodyParser.json(),
+    validateRequest,
+    graphqlExpress((req, res) => ({
+        schema,
+        context: () => {
+            const jwt = req.headers.authorization || '';
+
+            const token = getToken(jwt);
```

Comment:
could be `const token = verify(jwt.split(' ')[1], process.env.JWT_SECRET, { algorithms: ['HS512'] })`
Also `split(' ')[1]` does not take into account if token is empty